src/runtime/pkg/sev/vcpu_sigs.go

```diff
@@ -46,3 +46,31 @@ const (
 	// 'EPYC-Milan-v2': family=25, model=1, stepping=1
 	SigEpycMilanV2 VCPUSig = 0xa00f11
 )
+
+// NewVCPUSig computes the CPU signature (32-bit value) from the given family,
+// model, and stepping.
+//
+// This computation is described in AMD's CPUID Specification, publication #25481
+// https://www.amd.com/system/files/TechDocs/25481.pdf
+// See section: CPUID Fn0000_0001_EAX Family, Model, Stepping Identifiers
+func NewVCPUSig(family, model, stepping uint32) VCPUSig {
```

Comment:
So this is mainly to support future models given that we already have the existing sigs hard-coded? Seems like we could maybe remove some of the hard-coded ones given that we only use 3 for testing.

Reply:
The main use for this is when `cpuModel=="host"` (which is most cases for Kata), then we ask the host (using cpuid) for its family/model/stepping and use this function to convert to a VCPUSig.

The list of all the VCPUSig items is there to mimic the list of supported ones in QEMU; considering maybe some people will use this `pkg/sev` as a standalone go package in their programs.